COPYRIGHT.txt

```diff
@@ -126,8 +130,6 @@ Files: platform/android/java/editor/src/main/java/com/android/*
  platform/android/java/lib/src/main/aidl/com/android/*
  platform/android/java/lib/src/main/res/layout/status_bar_ongoing_event_progress_bar.xml
  platform/android/java/lib/src/main/java/com/google/android/*
- platform/android/java/lib/src/main/java/org/godotengine/godot/input/InputManagerCompat.java
- platform/android/java/lib/src/main/java/org/godotengine/godot/input/InputManagerV16.java
```

Comment:
These files don't seem to exist anywhere.

Reply:
They were removed in 22ca5b2ebac8d1756d46b9cf194cd1dc50ff2a86.